templates/CONTRIBUTING.md

```diff
@@ -0,0 +1,164 @@
+# How to Contribute
+
+We want your help to make Samvera great. There are a few guidelines
```

Comment:
I modified this by hand as "make the Samvera community great" elsewhere. What do you think?

Reply:
I just chose this because it's what's in Hyrax - do we want to focus on the community or the software here?